R/pkg/R/functions.R

```diff
@@ -3974,3 +3974,23 @@ setMethod("grouping_id",
             jc <- callJStatic("org.apache.spark.sql.functions", "grouping_id", jcols)
             column(jc)
           })
+
+#' input_file_name
+#'
+#' Creates a string column for the file name of the current Spark task.
+#'
+#' @rdname input_file_name
+#' @name input_file_name
+#' @aliases input_file_name,missing-method
```

Comment:
actually, could you add `@family normal_funcs` here? I missed this earlier and in the other PR.